app/services/platforms/facebook.ts

```diff
@@ -108,15 +109,16 @@ export class FacebookService extends StatefulService<IFacebookServiceState>
     return this.state.activePage.access_token;
   }
 
-  getHeaders(token = this.oauthToken): Headers {
-    const headers = new Headers();
-    headers.append('Content-Type', 'application/json');
-    headers.append('Authorization', `Bearer ${token}`);
-    return headers;
+  getHeaders(req: IPlatformRequest, useToken: boolean | string) {
```

Comment:
not sure why request is bring sent in here-- we dont seem to use it at all?

Reply:
It's added for compatibility with `IPlatformService`. For twitch API, depending on url, we use different authorization headers.